### ISSUE TITLE
Auf der N-IX weathermap....

### ISSUE THREAD

**Issue by ChristianDresel** [RedDog99/vortrag-erlug2018#1]
... fehlt noch F3 Netze e.V. Das muss unbedingt beim nächsten halten des Vortrags korrigiert sein ;)

**Comment by RedDog99** [comment]
Ja, die Weathermap wurde leider erst kurz nach dem Vortrag aktualisiert. Das Bild entspricht aber dem Stand der damaligen Zeit und ist somit leider richtig. Ein nächster Vortrag müsste natürlich aktualisiert werden.
